### ISSUE TITLE
how to fix the result ，even though I've set seed for everything

### ISSUE THREAD

**Issue by Tgzz666** [volcengine/verl#997]

**Comment by eric-haibin-lin** [comment]
Could u share what your current seeding code looks like, and which model you're using?

**Comment by Tgzz666** [comment]
I used the seed_everything function in main_ppo.py:
def seed_everything(seed: int):
    import random
    import os
    import numpy as np
    import torch

    random.seed(seed)
    os.environ["PYTHONHASHSEED"] = str(seed)
    np.random.seed(seed)
    torch.manual_seed(seed)
    torch.cuda.manual_seed(seed)
    torch.backends.cudnn.deterministic = True
    torch.backends.cudnn.benchmark = False
seed_everything(123)
Then I turned off shuffling, but I'm still getting inconsistent results each time. Is this a vLLM issue, or is it because of temperature=1.0?

**Comment by Kirrito-k423** [comment]
定时清理检查结果：这个 issue 最近一次由当前运行账号以外账号产生的公开回复或公开引用是 `2025-05-23T18:50:47Z`。本轮检查日期为 2026-07-04，该时间已经超过 6 个月。检查来源包括 issue comments API `https://api.github.com/repos/verl-project/verl/issues/997/comments?per_page=100` 和 timeline API `https://api.github.com/repos/verl-project/verl/issues/997/timeline?per_page=100`。

按照本仓库的定时 issue 清理标准，我会先留下这条说明并关闭该 issue。若作者或社区成员仍能提供新的复现、证据或需求背景，请继续评论请求重开，或创建新的 issue 并引用这里的上下文。

---

Scheduled cleanup check: the latest public reply or public reference from an account other than the current automation account was `2025-05-23T18:50:47Z`. The check date for this run is 2026-07-04, so that activity is more than 6 months old. The checked public sources were the issue comments API `https://api.github.com/repos/verl-project/verl/issues/997/comments?per_page=100` and the timeline API `https://api.github.com/repos/verl-project/verl/issues/997/timeline?per_page=100`.

Following the scheduled issue cleanup policy, I am leaving this note first and then closing the issue. If the author or community can still provide a fresh reproduction, evidence, or requirement context, please comment to request reopening or open a new issue that links back to this one.

This response was generated by a scheduled Codex GPT5.5 issue processing task.
